include/osquery/events.h

```diff
@@ -131,18 +133,22 @@ extern const std::vector<size_t> kEventTimeLists;
  */
 struct Subscription {
  public:
-  /// An EventPublisher%-specific SubscriptionContext.
```

Comment:
When you add `///` it becomes a shorthand doc-block. We're using it in header files for short descriptions attached to member variables or extremely obvious methods. Please replace `/`